util/net.go

```diff
@@ -28,6 +28,7 @@ import (
 
 	"golang.org/x/net/http2"
 
+	clog "github.com/cockroachdb/cockroach/util/log"
```

Comment:
heh, why leave this import aliased to `clog`? This just increased the size of the diff and resulting merge conflict.